fastly/bigquery_test.go

```diff
@@ -0,0 +1,295 @@
+package fastly
+
+import (
+	"testing"
+)
+
+func TestClient_Bigqueries(t *testing.T) {
```

Comment:
For me it feels like the test is trying to do too much all in one go.  I know that this is the style within the project and so just commenting.

Reply:
I totally agree! However, I don't want to split up the tests in this PR. I think it makes sense to discuss together how we want to refactor the tests and then change them all in a separate PR.

Reply:
Sounds good to me.